v3/src/components/app.tsx

```diff
@@ -100,6 +101,17 @@ export const App = observer(function App() {
           addDefaultComponents()
         }
       }
+
+      if (typeof urlParams.di === "string") {
+        const plugin = createDefaultTileOfType(kWebViewTileType)
+        const row = appState.document.content?.getRowByIndex(0)
+        if (isWebViewModel(plugin?.content) && row) {
+          plugin.content.setUrl(urlParams.di)
+          const pluginOptions = isFreeTileRow(row) ? { x: 5, y: 5 } : undefined
+          appState.document.content?.insertTileInRow(plugin, row, pluginOptions)
```

Comment:
The `di` parameter can be added when launching with existing documents as well, so my inclination would be to use something like `DocumentContent.createTile()` which will make use of the auto-placement code.

Reply:
I was able to get this to work, but I had to wrap it in a `setTimeout` because elements must be rendered for `getPositionOfNewComponent` to work. Is that alright? Is there another place it would make sense to add the plugin instead, so a `setTimeout` isn't necessary?

Reply:
I think the `setTimeout` is okay for now.